react/index.js

```diff
@@ -7,6 +7,7 @@ export { default as Footer } from './Footer/Footer';
 export { default as Header } from './Header/Header';
 export { default as Loader } from './Loader/Loader';
 export { default as Logo } from './Logo/Logo';
+export { default as RainbowLogo } from './RainbowLogo/RainbowLogo';
```

Comment:
Can we call this `Logo_Rainbow` to make the alphabetical ordering a bit cleaner?